rootston/cursor.c

```diff
@@ -310,8 +310,8 @@ void roots_cursor_handle_motion(struct roots_cursor *cursor,
 	double dx = event->delta_x;
 	double dy = event->delta_y;
 
-	wlr_seat_pointer_notify_relative_motion(cursor->seat->seat, (uint64_t)
-		event->time_msec, dx, dy, dx, dy);
+	wlr_seat_pointer_notify_relative_motion(cursor->seat->seat,
+		(uint64_t) event->time_msec, dx, dy, dx, dy);
```

Comment:
Extra space here

Reply:
I kinda find them dirty, but eh, your call. Formatting's all done. Let me add an example before merging.

Reply:
There's no need to add an example if there's already a weston example that does the same thing.